src/linear_samplers/block_col_rand_replace.jl

```diff
@@ -0,0 +1,44 @@
+# This code was Written by Nathaniel Pritchard
+"""
+    LinSysVecBlkColReplace <: LinSysBlkColSampler 
+
+A mutable structure with fields to handle randomly subset block sampling with replacement, also known as
+vanilla block randomized kaczmarz. 
```

Comment:
If this is for column sampling, then do not refer to it as Kaczmarz. This is not the right terminology.

Reply:
This was a holdover from copying the structure from the row version; I have corrected this.